config/default/language/fr/views.view.search.yml

```diff
@@ -4,7 +4,7 @@ display:
       empty:
         area:
           content:
-            value: '<p>No results matched your search.</p>'
+            value: '<p><span style="-webkit-text-stroke-width:0px;background-color:rgb(255, 255, 255);color:rgb(31, 35, 40);display:inline !important;float:none;font-family:-apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, &quot;Noto Sans&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;;font-size:14px;font-style:normal;font-variant-caps:normal;font-variant-ligatures:normal;font-weight:400;letter-spacing:normal;orphans:2;text-align:start;text-decoration-color:initial;text-decoration-style:initial;text-decoration-thickness:initial;text-indent:0px;text-transform:none;white-space:normal;widows:2;word-spacing:0px;">La recherche n’a produit aucun résultat.</span></p>'
```

Comment:
While the text content here looks correct, there appears to be lots of unnecessary code. Is this needed?